API_DOCUMENTATION.md

```diff
@@ -0,0 +1,2 @@
+alexgoz API Documentation:
+https://documenter.getpostman.com/view/19187581/UVsLRkx1
```

Comment:
don't forget to save another sample response

Reply:
alrigth, Sir, I will post another example